modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/PathsProcessor.java

```diff
@@ -287,6 +277,27 @@ protected boolean isLocalRef(String ref) {
         }
         return false;
     }
+    
+    protected boolean isAbsoluteRef(String ref) {
+    	 if(ref.startsWith("./")) {
+             return true;
+         }
```

Comment:
I'm sorry for asking a long time after this was merged, but wouldn't this make it a relative ref instead of an absolute one? Wouldn't it need to start with a `<scheme>:` to be considered [absolute](https://datatracker.ietf.org/doc/html/rfc3986#section-4.3)?

Reply:
You are right this is not correct.  
My intention was to have the isAbsoluteRef method match anything that is not relative.
The code should  then look like this:
protected boolean isAbsoluteRef(String ref) { return !ref.startsWith("."); ​}

protected String computeRelativeRef(String ref, String prefix) {
if(ref.startsWith("./")) {
​return ref;
​}
int iIdxOfSlash = prefix.lastIndexOf('/');
if(iIdxOfSlash != -1) {
return prefix.substring(0, iIdxOfSlash+1) + ref;
}
return prefix + ref;
​}